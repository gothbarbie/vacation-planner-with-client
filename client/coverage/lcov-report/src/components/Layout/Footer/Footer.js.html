<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/Layout/Footer/Footer.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">src/components/Layout/Footer</a> Footer.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">// @flow
&nbsp;
import React from 'react'
&nbsp;
import './Footer.css'
&nbsp;
import TextLink from '../../TextLink'
import Icon from '../../Icon'
&nbsp;
const Footer = () =&gt; (
  &lt;footer&gt;
    &lt;div className="footer__top" /&gt;
    &lt;div className="footer__main"&gt;
      &lt;div className="footer__inner footer__grid"&gt;
        &lt;div className="footer__author"&gt;
          Hanna Söderström &lt;br /&gt;
          info@hannasoderstrom.com
        &lt;/div&gt;
&nbsp;
        &lt;ul&gt;
          &lt;li&gt;
            &lt;TextLink url="https://github.com/gothbarbie/"&gt;
              &lt;Icon isBrand marginRight name="github" size="lg" /&gt;@gothbarbie84
            &lt;/TextLink&gt;
          &lt;/li&gt;
        &lt;/ul&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  &lt;/footer&gt;
)
&nbsp;
export default Footer
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Mar 11 2018 12:53:11 GMT+0100 (CET)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
