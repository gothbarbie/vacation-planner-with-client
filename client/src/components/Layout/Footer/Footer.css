.footer__top {
  background: linear-gradient(180deg, #3B3B3B 0%, #242424 100%), #4F4F4F;
  height: 20px;
}

.footer__main {
  height: 240px;
  background: linear-gradient(180deg, #3B3B3B 0%, #242424 23.76%), #4F4F4F;
  display: flex; 
  justify-content: center;
}

.footer__inner {
  width: 100%;
  max-width: 1140px;
  display: flex;
  flex-direction: row;
  padding: .5rem;
  color: #fff;
  font-weight: 100;
}
