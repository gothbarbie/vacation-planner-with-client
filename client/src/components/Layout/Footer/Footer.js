import React from 'react'

import './Footer.css'

import TextLink from '../../TextLink'
import Icon from '../../Icon'

const Footer = () => (
  <footer>
    <div className="footer__top" />
    <div className="footer__main">
      <div className="footer__inner footer__grid">
        <div className="footer__author">
          Hanna Söderström <br />
          info@hannasoderstrom.com
        </div>

        <ul>
          <li>
            <TextLink url="https://github.com/gothbarbie/">
              <Icon isBrand marginRight name="github" size="lg" />@gothbarbie84
            </TextLink>
          </li>
        </ul>
      </div>
    </div>
  </footer>
)

export default Footer
