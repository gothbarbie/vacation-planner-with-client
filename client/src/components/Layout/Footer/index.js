import React from 'react'

import './Footer.css'

import TextLink from '../../common/TextLink'
import Icon from '../../common/Icon'

export default () => (
  <footer>
    <div className="footer__top" />
    <div className="footer__main">
      <div className="footer__inner">
        <ul>
          <li>
            <TextLink url="https://github.com/gothbarbie/">
              <Icon isBrand name="github" size="lg" />@gothbarbie84
            </TextLink>
          </li>
        </ul>
      </div>
    </div>
  </footer>
)
