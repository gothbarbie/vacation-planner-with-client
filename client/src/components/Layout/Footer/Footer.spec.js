import React from 'react'
import { shallow } from 'enzyme'

import Footer from './Footer'

describe('components/Layout/Footer', () => {
  const footer = shallow(<Footer />)

  it('renders', () => {
    expect(footer).toMatchSnapshot()
  })
})
