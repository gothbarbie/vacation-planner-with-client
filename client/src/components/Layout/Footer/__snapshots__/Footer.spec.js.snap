// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Layout/Footer renders 1`] = `
<footer>
  <div
    className="footer__top"
  />
  <div
    className="footer__main"
  >
    <div
      className="footer__inner footer__grid"
    >
      <div
        className="footer__author"
      >
        Hanna Söderström 
        <br />
        info@hannasoderstrom.com
      </div>
      <ul>
        <li>
          <TextLink
            url="https://github.com/gothbarbie/"
          >
            <Icon
              isBrand={true}
              marginRight={true}
              name="github"
              size="lg"
            />
            @gothbarbie84
          </TextLink>
        </li>
      </ul>
    </div>
  </div>
</footer>
`;
